src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -180,6 +181,16 @@ def failure_count_transaction(org_id, metric_ids, alias=None):
     )
 
 
+def failure_rate_transaction(failure_count_snql, tx_count_snql, alias=None):
```

Comment:
I would rename this to just `divide` because its practically doing that and not specific to transactions... See line 194, 198, 202

Reply:
I thought about that, but I don't really like it. I think a more explicit function name considering the context where it's used makes refactoring and adding stuff on top of it easier. That said, thanks a lot for this feedback, especially on these small improvements.

Reply:
I disagree on this.. this basically breaks the consistency of the entire module